### ISSUE TITLE
2.7.0-dev (de)Bug?: Config Log-Level ineffectual.

### ISSUE THREAD

**Issue by thebeline** [Z-Bolt/OctoScreen#253]
For whatever reason, if I edit the config from `Debug` to `Info`, and the `debug` logs still seem to keep coming.  Not sure if this is `2.7.0` exclusive or a previous issue, but restarting the service, and the pi, do not resolve this.

Additionally, as I have been developing, I have removed and installed completely new builds several times, and the debug logs are still coming...

**Comment by thebeline** [comment]
Related??? - It would seem that I actually over-wrote my config file and the API Key has been cleared...  But...  It is still connecting...

Um...  Huh?

**Comment by thebeline** [comment]
A quick `git grep`:
```
pi@octopi-ctc:~/OctoScreen-2 $ git grep OCTOSCREEN_LOG_LEVEL
README.md:- `OCTOSCREEN_LOG_LEVEL` - Controls the level of logging.  Accepted values are (with increasing levels): `debug`, `info`, `warn`, and `error`.  If no value is provided, the log level will default to `warn`.
debian/local/octoscreen/config:OCTOSCREEN_LOG_LEVEL=Error
utils/environment.go:   EnvLogLevel    = "OCTOSCREEN_LOG_LEVEL"
pi@octopi-ctc:~/OctoScreen-2 $ git grep EnvLogLevel
main.go:                        os.Setenv(utils.EnvLogLevel, "error")
utils/environment.go:   EnvLogLevel    = "OCTOSCREEN_LOG_LEVEL"
utils/environment.go:   dumpEnvironmentVariable(EnvLogLevel)
```

I am not sure about the API key, but it certainly looks like the Log Level isn't even referenced...

**Comment by thebeline** [comment]
Resolved with PR #254
